components/eam/src/physics/cam/physics_types.F90

```diff
@@ -1587,6 +1591,9 @@ subroutine physics_state_alloc(state,lchnk,psetcols)
   
   allocate(state%pv(psetcols,pver), stat=ierr)
   if ( ierr /= 0 ) call endrun('physics_state_alloc error: allocation error for state%pv')
+
+  allocate(state%pt(psetcols,pver), stat=ierr)
+  if ( ierr /= 0 ) call endrun('physics_state_alloc error: allocation error for state%pv')
```

Comment:
Message should use `state%pt` instead of `state%pv`.

Reply:
agreed; fixed